src/sagemaker/jumpstart/payload_utils.py

```diff
@@ -115,80 +115,6 @@ def _construct_payload(
     return payload_to_use
 
 
-def _extract_generated_text_from_response(
```

Comment:
nit: Could you specify why you are removing this in the PR description?

Reply:
sorry, will do